src/javascripts/components/blog/ArticlePreview.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { Link } from 'react-router'
+
+class ArticlePreview extends React.Component {
+  render() {
+    return (
+      <Link to="/article"><div className="articlePreview">
+        <div className="--imageContainer">
+          <img
+            src="./images/alice-in-wonderland.png"
+            alt="Picture of 'Author'"
+          />
+        </div>
+          <div className="--preview">
+            <h1>Down The Rabbit Hole</h1>
+            <div className="articlePreview-postMeta">
+              <small className="articlePreview-authorName">
+                <Link to="/author">Lewis Carroll</Link>
+              </small>
```

Comment:
`<small>` is typically use for legal text, aka "fine print".  Use CSS to make the text smaller.

Also -- you've got an `<a>` (or LInk) inside of an `<a>` which is... kinda gross.  You might want to re-work some HTML here to avoid that.